src/mat/4C_mat_micromaterial.hpp

```diff
@@ -33,6 +34,13 @@ namespace Mat
       /// standard constructor
       MicroMaterial(const Core::Mat::PAR::Parameter::Data& matdata);
 
+      //! action parameters recognized by ale3
+      enum class runtime_output_option
```

Comment:
Wrong case style. 

```suggestion
      enum class RuntimeOutputOption
```

Reply:
Also, please use this enum directly in the `add_named_selection_component` call I mentioned above.

Reply:
I changes the case style. How can I used this in the `add_named_selection_component` ?

Reply:
See e.g. here: https://github.com/4C-multiphysics/4C/blob/28fa9e3e9ae1d5e42f29814321560d1336278ee3/src/inpar/4C_inpar_ssi.cpp#L351-L354